### PR TITLE
Mat 5535 qdm scoring unit

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-5535](https://jira.cms.gov/browse/MAT-5535)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx [hunk 1]
```diff
@@ -526,6 +524,7 @@ const MeasureGroups = (props: MeasureGroupProps) => {
           });
         });
     } else {
+      group.populations[0].id = uuidv4();
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: Assigning id to the first population, what does this do?

**Review by adongare — DISMISSED** [review]

File: src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: @adongare This is when adding a group, id is required. Thanks Ashok!

Comment (adongare) [reply src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: ignore my lack of QDM understanding. Haven’t reviewed earlier PRs. is it not required for other populations?

Comment (sb-cecilialiu) [reply src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: @adongare No you are right it should be for all populations, I just changed it, thanks so much Ashok!

**Review by adongare — APPROVED** [review]

**Pull request merged**